pkgdb/src/buildenv/assets/etc/profile.d/0100_common-paths.sh

```diff
@@ -57,22 +77,21 @@ export MANPATH
 
 # ---------------------------------------------------------------------------- #
 
-
 if [ -n "${FLOX_ENV_LIB_DIRS:-}" ]; then
   case "$(uname -s)" in
-  Linux*)
-    # N.B. ld-floxlib.so makes use of FLOX_ENV_LIB_DIRS directly.
-    if [ -z "${FLOX_NOSET_LD_AUDIT:-}" -a -e "$LD_FLOXLIB" ]; then
-      LD_AUDIT="$LD_FLOXLIB";
-      export LD_AUDIT;
-    fi
-    ;;
-  Darwin*)
-    if [ -z "${FLOX_NOSET_DYLD_FALLBACK_LIBRARY_PATH:-}" ]; then
-      DYLD_FALLBACK_LIBRARY_PATH="$FLOX_ENV_LIB_DIRS:${DYLD_FALLBACK_LIBRARY_PATH:-/usr/local/lib:/usr/lib}";
-      export DYLD_FALLBACK_LIBRARY_PATH;
-    fi
-    ;;
+    Linux*)
+      # N.B. ld-floxlib.so makes use of FLOX_ENV_LIB_DIRS directly.
+      if [ -z "${FLOX_NOSET_LD_AUDIT:-}" -a -e "$LD_FLOXLIB" ]; then
```

Comment:
```suggestion
      if [ -z "${FLOX_NOSET_LD_AUDIT:-}" ] && [ -e "${LD_FLOXLIB?}" ]; then
```

Reply:
Why incur the extra exec?
```
flox [flox/prerelease default] [brantley@saas:~]$ strace -f bash -c '[ -z "${FLOX_NOSET_LD_AUDIT:-}" -a -e "$LD_FLOXLIB" ]' |& wc
    179    1290   17109

flox [flox/prerelease default] [brantley@saas:~]$ strace -f bash -c '[ -z "${FLOX_NOSET_LD_AUDIT:-}" ] && [ -e "${LD_FLOXLIB?}" ]' |& wc
    193    1411   18721

flox [flox/prerelease default] [brantley@saas:~]$ strace -f bash -c '[ -z "${FLOX_NOSET_LD_AUDIT:-}" ] && [ -e "${LD_FLOXLIB?}" ]' |& grep -i exec | wc
     29     247    4170

flox [flox/prerelease default] [brantley@saas:~]$ strace -f bash -c '[ -z "${FLOX_NOSET_LD_AUDIT:-}" -a -e "$LD_FLOXLIB" ]' |& grep -i exec | wc
     28     242    4042
```
I'd vote no on that basis.

Reply:
Sh builtin/test portability across distros

Reply:
For testing and most other contexts where we control the runtime env `-a` and `-o` are fine just for clarity.

Its just in these activation scripts that we care

Reply:
Our activation scripts ought to be sourced by _our_ choice of shell prior to exec'ing the user's $SHELL. It's just too big an error vector to expect every $SHELL to be posix compliant (says the guy who was still using tcsh circa 2018).

Reply:
Don't we have to support $SHELL because of aliases/rc files etc?